src/wasm/client.rs

```diff
@@ -284,15 +341,27 @@ impl Default for ClientBuilder {
     }
 }
 
-#[derive(Clone, Debug)]
+#[derive(Clone)]
 struct Config {
     headers: HeaderMap,
+    #[cfg(feature = "cookies")]
+    cookie_store: Option<Arc<dyn cookie::CookieStore>>,
+}
+impl std::fmt::Debug for Config {
+    fn fmt(&self, f: &mut std::fmt::Formatter) -> std::fmt::Result {
+        f.debug_struct("Config")
+            .field("headers", &self.headers)
+            .field("cookie_store", &self.cookie_store.is_some())
```

Comment:
Seems this fails in CI?

Reply:
Oh, yes, if cookies aren't enabled then this line fails because cookie_store is not a field. Sorry about that - I modified my rust-analyzer settings locally to always enable cookies and compile for wasm32.